db/views.h

```diff
@@ -261,12 +270,27 @@ int timepart_is_shard(const char *name, int lock);
 int timepart_is_timepart(const char *name, int lock);
 
 /**
- * Alter a timepart 
+ * Time partition schema change resume
+ *
+ */
+int timepart_resume_schemachange(int check_llmeta(const char *));
+
+/**
+ * During resume, we need to check at if the interrupted alter made any
+ * progress, and continue with that shard
+ *
+ */
+int timepart_schemachange_get_shard_in_progress(const char *view_name,
```

Comment:
same as above. Why do we need this? I think `resume_schema_change` already loops through all tables in the `thedb->num_dbs` array.

Reply:
I need these for partition resume, it is not done.

Reply:
I think resume_schema_change already takes care of resuming multiple schema changes?